PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/PBMPrebidParameterBuilder.m

```diff
@@ -178,6 +178,7 @@ - (void)buildBidRequest:(nonnull PBMORTBBidRequest *)bidRequest {
         nextImp.extPrebid.storedAuctionResponse = Prebid.shared.storedAuctionResponse;
         nextImp.extPrebid.isRewardedInventory = self.adConfiguration.adConfiguration.isOptIn;
         nextImp.extGPID = self.adConfiguration.gpid;
+        nextImp.extOrtbObject = self.adConfiguration.ortbObject;
```

Comment:
Hi @jsligh , @bretg !

I would like to clarify the scope of the implementation to provide the proper review. 

For now, I see support only for the `imp.ext` field. So, the publisher can set the value of this by providing the respective JSON. 

However, from my understanding of the feature request, this new API should provide the ability to set _any_ part of the request with the given objects. 

For instance, the publisher should be able to set the whole imp object like: 

```
adUnit.setOrtb("
{
 "imp": [
    {
      "id": "1",
      "bidfloor": 0.03,
      "banner": {
        "h": 250,
        "w": 300,
        "pos": 0
      }
    }
  ]
}
")
```

Or set the app info like: 

```
adUnit.setOrtb("
{
"app": { "id": "agltb3B1Yi1pbmNyDAsSA0FwcBiJkfIUDA", "name": "Yahoo Weather", "cat": [ "IAB15", "IAB15-10" ], "ver": "1.0.2", "bundle": "12345", "storeurl": "https://itunes.apple.com/id628677149", "publisher": { "id": "agltb3B1Yi1pbmNyDAsSA0FwcBiJkfTUCV", "name": "yahoo", "domain": "www.yahoo.com" } },
})
```

Note: these examples are copy-pasts from the Opertb doc just for demo purposes. Don't treat them as an implementation spec.

So, if the request is about the second variant, the implementation should be extended.

Reply:
You're right @YuriyVelichkoPI that the goal is for the SDK to be able to supply any arbitrary ORTB parameter.

 My suggestion in the original issue was to break this into two parts: the "global" part and the "imp-level" part:
```
   addOrtbGlobal('{"ext":{"prebid":{"debug":1,"trace":"verbose"}}}');
   adUnit = BannerAdUnit(configId: "my-stored-request", size: adSize, "{instl:1, ext:{ gpid: "222", data: { foo: "bar" }})
```

I don't see them every wanting to supply the entire `imp` object, but they need to be able to add any field under imp, not just in imp.ext.

Reply:
The global `ext` object (and any other global params) would be set in a new function, e.g. `addOrtbGlobal`